web-components/button/button.svelte

```diff
@@ -22,16 +42,36 @@
   }
 </script>
 
+<svelte:element
```

Comment:
I have no idea why but the formatter in my VSCode really wants it to be this way. If you think the approach is reasonable I'll put this back to the way it was.